packages/pug/lib/index.js

```diff
@@ -303,6 +305,7 @@ exports.compileClientWithDependenciesTracked = function(str, options){
     filterOptions: options.filterOptions,
     filterAliases: options.filterAliases,
     plugins: options.plugins,
+    module: options.module
   });
 
   return {body: parsed.body, dependencies: parsed.dependencies};
```

Comment:
I need help here:
`templateName: options.name || 'template',`
I don't understand why `options.name` is not `options.templateName`
Thanks! :)